src/main/java/io/github/thebusybiscuit/cscorelib2/protection/modules/FunnyGuildsProtectionModule.java

```diff
@@ -0,0 +1,48 @@
+package io.github.thebusybiscuit.cscorelib2.protection.modules;
+
+import io.github.thebusybiscuit.cscorelib2.protection.ProtectableAction;
+import io.github.thebusybiscuit.cscorelib2.protection.ProtectionModule;
+import me.angeschossen.lands.api.integration.LandsIntegration;
+import me.angeschossen.lands.api.land.Area;
+import me.angeschossen.lands.api.land.LandWorld;
+import me.angeschossen.lands.api.role.enums.RoleSetting;
+import net.dzikoysk.funnyguilds.FunnyGuilds;
+import net.dzikoysk.funnyguilds.system.protection.ProtectionSystem;
+import org.bukkit.Location;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.Plugin;
+
+public class FunnyGuildsProtectionModule implements ProtectionModule {
```

Comment:
Could you please add some javadocs to this class, with a short description and an `@author` tag so we know where this comes from.